test/e2e/test-data/otelcollector.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: v1
```

Comment:
where are we using this file?

Reply:
working on the e2e test using this file, wanted to push the change of the deployment